api/ws/server.go

```diff
@@ -33,16 +34,25 @@ var (
 	ErrExpired = errors.New("expired")
 )
 
-func WithWebsocketAPI(maxPendingMessages int) vm.Option {
-	return func(v *vm.VM) error {
+type WSConfig struct {
+	MaxPendingMessages int `json:"maxPendingMessages"`
+}
+
+func WithWebsocketAPI() vm.Option {
+	return func(v *vm.VM, r json.RawMessage) error {
```

Comment:
Might want to name `r` `configBytes` or something

Reply:
done